apiBase/errors/validationError.es6.js

```diff
@@ -1,13 +1,19 @@
 import FakeError from './FakeError';
 
+const msgText = (apiName, errors, model) =>
+  `${ apiName } had errors in ${ errors.join(',') } with properties ${ JSON.stringify(model.toJSON()) }`;
```

Comment:
🔕 I would wrap this in a eslint disable comment for line length. Not that it super matters with all the other linter errors at the moment ...